### PR TITLE
adding link to Google maps in the facility modal

### PR DESCRIPTION
**Pull request by emre2038**
## Description
adding a view location in Google Maps link in the facility modal. 
https://github.com/OpenUpSA/wazimap-ng-ui/pull/271

## Related Issue
https://trello.com/c/4ZXzJiCT/641-add-link-to-google-maps-in-the-facility-modal

## How to test it locally
* on the point mapper click on any category
* wait until the point markers are created on the map
* click on a point marker 
* on the tooltip click on `Show More` button to view the facility modal
* confirm that in the facility modal, we now have the Google Maps link and it redirects to the correct coordinates

## Screenshots
image

### Updated
* `index.html` and `wazimap-ng-v1.webflow.css` to apply design changes
* `point_data.js ` to modifiy `href` property of the link button

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by emre2038** [comment]
[Add link to Google maps in the facility modal](https://trello.com/c/4ZXzJiCT/641-add-link-to-google-maps-in-the-facility-modal)

File: src/js/map/point_data.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ let tooltipRowItem = null;
 let facilityItem = null;
 let facilityRowItem = null;
 
+let googleMapsButton = null;
```
Comment (adieyal) [inline src/js/map/point_data.js:29]: Why are all these global variables?

Comment (emre2038) [reply src/js/map/point_data.js]: moved the variables into the class

**Comment by michaelglenister** [comment]
Works in deploy preview and I didn't find any unexpected issues

File: src/index.html [hunk 2]
```diff
@@ -738,6 +738,18 @@ <h3 class="facility-info__title">Loading...</h3>
             </svg></div>
         </div>
       </div>
+      <a href="#" class="facility-info__view-google-map hidden w-inline-block">
```
Comment (waracci) [inline src/index.html:741]: Was this change made on Webflow as well? If it wasn't, there is a risk it might be overwritten by future exports.

Comment (emre2038) [reply src/index.html:741]: Yes, this code was taken from a webflow export

**Comment by waracci** [comment]
@emre2038 There is pending work on this, could you make the link open in new tab as suggested by Adi here https://trello.com/c/4ZXzJiCT/641-add-link-to-google-maps-in-the-facility-modal#comment-60dd640f3b0e96092e4b57c4

**Pull request merged**